src/System.Runtime.CompilerServices.Unsafe/ver/System.Runtime.CompilerServices.Unsafe.Versioning.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0</TargetFrameworks>

```

Comment:
Nit: I think you only need `netstandard2.0`

Reply:
Hmm the ProjectReference is conidtioned to all except net7.0:
```xml
  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
    <ProjectReference Include="..\ver\System.Runtime.CompilerServices.Unsafe.Versioning.csproj"
                      PrivateAssets="All"
                      ReferenceOutputAssembly="false"
                      OutputItemType="ILResourceReference" />
```
Should we change that condition to only import the project for ns2.0?

Reply:
@ericstj No, I need NetFrameworkMinimum here. I built with with IsPackable=true, opened the DLLs for net462 using ILSpy, and with NetFrameworkMinimum included, the AssemblyVersion is correct (6.0.1.0), but it's incorrect if I remove NetFrameworkMinimum (6.0.0.0).

Reply:
I would only expect this to impact *file version* assembly version should be determined by the AssemblyVersion properties in the ilproj.  I would expect file version to be the same for everything.

The reason I said you could remove `NetFrameworkMinimum` is that the build of the ilproj for `NetFrameworkMinimum` can just use the `netstandard2.0` copy of the Versioning project.

Also -- I thought we were going to give this a different name like "private.version" or something to avoid accidentally shipping it.

Reply:
The name we choose for this versioning project is what shows up in the DLL Properties -> Details -> Product name. I thought we would prefer it to be named "System.Runtime.CompilerServices.Unsafe.Versioning" rather than seeing just "Version".

Reply:
Oh, that's interesting.  Nice catch!  In that case let's keep your project name, but lets make the assembly named "System.Runtime.CompilerServices.Unsafe" so that we get the right value in that field of the version properties.

Reply:
I'll try in a separate PR. If I recall correctly, using that same name caused build issues. Will let you know tomorrow.

Reply:
@ericstj do you have a strong opinion on the name this project should have?

Reply:
You can't have the same project name.  They fight over the output directory.  That's why I mentioned above to set `AssemblyName`
> In that case let's keep your project name, but lets make the assembly named "System.Runtime.CompilerServices.Unsafe"

Does this work?

Reply:
Oooooooh! I didn't know there was a property for that.

Reply:
https://learn.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2022#list-of-common-properties-and-parameters